services/web/.env

```diff
@@ -1,18 +1,29 @@
 # Note that changing the variables here will require a restart of the app server
 
-ENV_NAME=development
+##########################################################
+# Variables prefixec with SERVER or HTTP are not exposed #
+##########################################################
 
-BIND_HOST=0.0.0.0
-BIND_PORT=2200
-APP_NAME=Bedrock
-API_URL=http://localhost:2300
+SERVER_HOST=0.0.0.0
+SERVER_PORT=2200
+
+# Http force redirect to https
+HTTP_FORCE_HTTPS=
```

Comment:
maybe this is an opportunity to remove this flag altogether... #1 HSTS is now kind of the de-factor way to handle (vs 301 redirect) and #2 we tend to just let cloudflare layer handle this

Reply:
cool lets remove it